tutorials/fileToJdbc/.gitignore

```diff
@@ -0,0 +1,3 @@
+.DS_Store
+/.idea/
+/target/
```

Comment:
Please do not checkin artifacts generated by the build process such as 'target' directory